packages/fxa-content-server/server/config/local.json-dist

```diff
@@ -25,14 +25,12 @@
           "http://localhost:3030/oauth/success/a2270f727f45f648",
           "http://localhost:3030/oauth/success/1b1a3e44c54fbb58",
           "urn:ietf:wg:oauth:2.0:oob:pair-auth-webchannel",
-          "urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel",
```

Comment:
From here down is a correction of a change I made in #16706. The changes to `redirectUri` above are a better way to fix the problem I was experiencing.